components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/latest/GitpodPortForwardingService.kt

```diff
@@ -91,8 +91,9 @@ class GitpodPortForwardingService(private val project: Project) {
         for (port in response.portsList) {
             val hostPort = port.localPort
             val isServed = port.served
+            val isForwarded = forwardedPortsList.find { it.hostPort == hostPort } != null
 
-            if (isServed && !forwardedPortsList.containsKey(hostPort)) {
+            if (isServed && !isForwarded) {
```

Comment:
ℹ️ This was changed because `forwaredPortsList` became a List instead of a Map in v2022.3.